dockerfiles/Dockerfile.coordinator

```
@@ -44,8 +44,8 @@ COPY --from=build \
 
 # the coordinator container image
 FROM ubuntu:focal-20221130 AS release
```

```suggestion
FROM ubuntu:focal-20230308 AS release
```
also in Dockerfile.build-base